src/engraving/libmscore/hairpin.cpp

```diff
@@ -643,9 +643,9 @@ Hairpin::Hairpin(Segment* parent)
 {
     initElementStyle(&hairpinStyle);
 
+    resetProperty(Pid::HAIRPIN_TYPE);
```

Comment:
no actual reason to move this, but originally I had tried putting it before initElementStyle( ) to see if it helped (it didn't).